clang/test/CIR/CodeGen/agg-init3.cpp

```diff
@@ -0,0 +1,38 @@
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -std=c++17 -fclangir -emit-cir %s -o %t.cir
+// RUN: FileCheck --input-file=%t.cir %s
+
```

Comment:
Please rename the file to `array-init.cpp`, or if the expected output is the same between C and C++, then might be better just to append this to existing `clang/test/CIR/CodeGen/array-init.c` instead.